src/Web/Action/MusicPageAction.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Web\Action;
+
+use Slim\Http\Request;
+use Slim\Http\Response;
+use Slim\Views\PhpRenderer;
+use Web\Domain\SpotifyFeed;
+use SpotifyWebAPI\SpotifyWebAPI;
+use Slim\PDO\Database;
+
+class MusicPageAction
+{
+    /**
+     * @var array
+     */
+    private $renderer;
+    private $spotify;
+    private $db;
+
+    public function __construct(
+        PhpRenderer $renderer,
+        SpotifyWebAPI $spotify,
+        Database $db
+    ) {
+        $this->renderer = $renderer;
+        $this->spotify = $spotify;
+        $this->db = $db;
+    }
+
+    public function __invoke(Request $request, Response $response, $args)
+    {
+        // Verify if user is authenticated. If false, redirect to login
+        if (isset($_SESSION['user'])) {
```

Comment:
I think perhaps a perfect case for middleware -- is user authenticated?  If not, redirect to the login.